src/freenet/client/filter/CSSTokenizerFilter.java

```diff
@@ -1211,6 +1232,36 @@ else if("margin-bottom".equalsIgnoreCase(element))
 			elementVerifiers.put(element,new CSSPropertyVerifier(Arrays.asList("auto"),ElementInfo.VISUALMEDIA,Arrays.asList("le","pe")));
 			allelementVerifiers.remove(element);
 		}
+		else if("margin-block".equalsIgnoreCase(element))
+		{
+			elementVerifiers.put(element,new CSSPropertyVerifier(null,ElementInfo.VISUALMEDIA,null,Arrays.asList("80<1,2>")));
+			allelementVerifiers.remove(element);
+		}
+		else if("margin-block-end".equalsIgnoreCase(element))
+		{
+			elementVerifiers.put(element,new CSSPropertyVerifier(Arrays.asList("auto"),ElementInfo.VISUALMEDIA,Arrays.asList("le","pe")));
+			allelementVerifiers.remove(element);
+		}
+		else if("margin-block-start".equalsIgnoreCase(element))
+		{
+			elementVerifiers.put(element,new CSSPropertyVerifier(Arrays.asList("auto"),ElementInfo.VISUALMEDIA,Arrays.asList("le","pe")));
```

Comment:
Can this refer to `auxilaryVerifiers[36]` as `36` instead of hard-coding `length | percentage`?

(same for `margin-block-end`, similar for all the others padding/margin variants added)